_events/OSG-User-School-2023.md

```diff
@@ -18,14 +18,17 @@ header_image: "/images/Madison_Skyline.jpg"
 
 The OSG User School will occur on August 7–11, where you will learn to use high-throughput computing (HTC) systems to run large-scale computing applications at your campus or using the national-scale OSG.
```

Comment:
As the first content line of this page, could it be made more interesting and appealing?